src/ipc-handlers.ts

```diff
@@ -651,7 +651,10 @@ export async function exportSiteToPush( event: IpcMainInvokeEvent, id: string )
 		splitDatabaseDumpByTable: true,
 	};
 	// eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
@epeicher , Should we remove the eslint comment `// eslint-disable-next-line @typescript-eslint/no-empty-function`?